server_extensions_extra/src/main/resources/lunchmoney.svg

```diff
@@ -0,0 +1,9 @@
5RKGNkVcq/NrMBg5Xpdf+6YVPb8q4zAD9zUf1iclJKeEtZlI0hG4Fl0pX+uwzGnOICE1CxcbUqm0ex8yuxOEaMKQ40tJCFNWkuFSweVlhZmnqyuhrWfkrTxAJdAuVzY0iDkek1z7z9acnn70FyBga5t7Pffpx/faFMdCpmwhZiHTbbOZdMWYkwGGlGQWyd+Qb0fZc6jZPc9MNiN+C3RWpZXM8UGaz6tvMzRl7riDqHBge5jYTacxN2lgODP+Gv8B6+dl77rgFyMTc+457nj35CXTq6OpqkkZ6JSvnSFMT54px6UZjadkRFGbYAXOUVeuyszLLMzxw+66LXJOwOh89nr48IxSdNsovGbemc69dKTRuLirIzzhBpGrUCdEIyP77N3jx+ItbgYygYWk589k3oFNHJidVhx9B95D1gmtRTY1Nwm+N6zeb0rzwneVFuabyWrZQAM1MY8G5KmFxPE5M3nd2KQgDi3ZFrsmUv2rdJmP+Sg8UoUKD1DE979qVLLTFOd+MU4nQwkv/J+y9bhkyhiY69cf1eQvTq0s8cCvBDTU2ipCnKa3BAep9xs1lDXBgkHy7G7EuupRf5Y7Og92PMCH7R0VFu+Ej6C2lFWEBnGtXjCSfDPmupkYDH6iNq2ZB0RpdrDBWbdCBXWOHfUObX9cDDZ0a7FsRBap2o6YX6u+MBaZUL3SIGp1W2YiDOWz4q12SZSzzhrMlq1K3eWvo4h8cFY1DDHtW5S/1QEf6a1mby5To0n1XmJmBghpVaJBiwYFOfG1HBExp9KLYWFiK/Anb9T03e4yaQH077tRvfP0RoWpPX6Lxhu8KTeWzMKh2SaGpio6mu5ZZurIB0iC320BH1i9ZVZo23weO4DoZ0+JRGFhM+RxcEBqa0nE6KVI0DtMZoAKdCpbXOt0zX9rt+hYik55s5tNI1dsVUXHVce7EFJL/3ZUKX3wEaKUriUNCYhEFpAKa0sVKWoRWf62wIHPpXF+tel8m9FSUyJYaHWWWbsPpQdH5GSwyk0GkUonUOhUoGdn1revKdGhs38+eRKoWRJeUhHrDnBnaJQ7Qosm0VgeWuE2pt5MFXjGlxUPbuXsBS6unMys2FZjyq9I1guwCzIzKwaR6sqCVPji99ErmsmiUFRZeiFgtQkr+AezaRn27XdWREeklSZ6+eprorhWWrnSnBYhh3K6qmV0J07S0aL2DyC4Ba35GJV6ZIVTIDuuHDDe80EIFEUWrvtuZ4E+svfAE7k+Ekn8Au7ZBw6CFVP3G1x9v3y5PDi2J81AabKJHbnabmwY1EcuSQvTyjNLyNKNxqZeBLnrJlYL1qzJLl+ab0vaBUpU4O6wZ01FWNfIDGiKlUIVCYMqKJq2xyF1jJf8Adk2DJlR98suPtudJo2KA2s+gI0mE320yli+tmR1V6afDTc8qKM1FX5cqMJb56hygBVdMW1dcgZjSmmugXh8UC2smbCeGhatgimdPq9Pp1GDUothnXgB3jZT8A9k1jZpQNTLwRHEwSU0I77tLmaDYZVUh6Cz2O+PmnEXX9u2FcXelVkcXA+9bY9qcxCsmU2pw5CZjlRcKVEdegilYTe1GtN/m75BeLVLFPoPNmlLyD4hMqRomKG98Y6P2DDGocds4im8vFWblluEWc78FdelEKj445MUGBwaIAycd+92l0lkG+Sbj0micnQN2Yiw1wmRsI9t2dXRoYyEWiEmz/uF6sjP1L7GBf/MxUEuiQpOSQt38wf7UOjVHvGTJkgBkjDqw37RK+MtDf1GIXXTo4wvb/Lnf1fnrpJcQM5bA764UZmXlrl4hIlLhjUE7MWHWPwYyqWowcKD+GlFLAzzikmJYfiSXiGixWrbJtDQE/kovmVIr1Y4S+x30XX81h6NWi+GSRFPBgiVLFi0UEsQGPd4aFAqB+BtM/COYtRP1HdCtQdeffJaXKA+rjEmK8wRl4xEF/xYprmzK8Ud/C41Lg0UOokazygw/NN6q+ahrq8lwIj/8Q/tnep1IyOFQOv6xzJqB+qO8vEhxcnx4XLinl7+aAuNKloTBiepaYWaZv8hJwFtXheCjwG+z8her6XEYWAsq5nC4f3Eg/lGREbSN+o1Pv/wYKXthlEd4XIynmx9bhPb5RCJiv096bW+CWuUooshLmUuD8ZHw2r4lASIqXIS2jZQaPQBzA595/stvKOIfqyM7U99OUIMPP/k1KDsxUoaxw0NZIf5skQ1QyIlVOYv4u30r2OQeKBULCgZepRLWEgDMAxUj4mehH4OOfwrESAjqX86Y+TSY+Kdff/xRImDzk6M9w8PDPVlewUGiCay2zV7axi/CZev8/ZS+vr5Kf7aQGxjIe+bvhIp/YsRICAuHkQsr+5svMbZCEhYV7RkaHhrqAeBY47EMgmjZQf5+vj4gSi07lhPI4/H/gYCxipFR//j92FkoaqxswAYj/wC45ZKw5BB3j1AQTw+Wu1eIn79/UBAbBJwU+sMOAlS/EB8vb3RLx8fPXyTkAi5f8DwJDFOumT9JYiyIGmPfg7E/Rer+IDEyUiETBwYEhbixPDyxeICwsLiD4N9uiJatAu0CrUAMCkZ9mLLpXxJG/dMjRnKbIzZwf/3lJx9/8AFwK+TofgUngB0UHBLiRUpICOg9iB0gRLACgBVL/vE85kUKpoAJFf80iZHcRvgzbOQU9zcI/KMPPlCA4Js1EolELBYDISESLP/4x/MYF+sXKfienwUwIQQ21raN++SniBzQP37++Q8++OAfdHkeWD/5EtMCLqFf4KUB/9SJsVDYFDeAY3LE/uk333zztU3gBFAJ2DcwLsmL+/DPBpiQ22jcGByRAzqCx3KSwHwDoz737LOIFuNi/QLvzwyYFIobgWPyGTOBHdHTBYXMBFhEi3F/tryUENwIHMgBHdixzMBCnvwSwSJaCvfny0vJbRQ5wU7yI8pfUOcIlqD9+ePShCC30dvkNrv82E28ZXKbi58fVv4fqy+taiD89b4AAAAASUVORK5CYII="/>
```

Comment:
Raster image is not supported inside SVG. Use only vectors inside SVG.

Reply:
Thank you. We'll resubmit the logo in the proper format.

Reply:
updated svg